videos/videos.js

```diff
@@ -31,94 +31,111 @@ router.get("/:id", async (req, res) => {
 // videos: video_title, location
 // We'll want to adjust the model to allow for both
 
-
 router.post("/", restricted, async (req, res) => {
   // this should be unnecessary because we're rebuilding objects
```

Comment:
We can probably remove these older comments, since they'd be confusing to someone getting recently onboarded